log_emitter.go

```diff
@@ -0,0 +1,32 @@
+package composer
```

Comment:
I think you can get rid of this entire file. the `scribe.Emitter` type in packit already contains these functions which can be used instead.

Reply:
done